pkg/config/setup/config.go

```diff
@@ -472,6 +472,10 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("cluster_agent.language_detection.cleanup.period", "10m")
 	config.BindEnvAndSetDefault("cluster_agent.kube_metadata_collection.enabled", false)
 	config.BindEnvAndSetDefault("cluster_agent.kube_metadata_collection.resources", []string{})
+	config.BindEnvAndSetDefault("cluster_agent.kube_metadata_collection.resource_annotations_exclude", []string{
```

Comment:
if we want to make it visible to customer we should document it in the file 
https://github.com/DataDog/datadog-agent/blob/836e82bfa8f595a6f48d04a30a4d8bfe2add47d9/pkg/config/config_template.yaml

something that we need to be careful is that if we refactor auto-discovery on (service/endpoint) we can't remove the the `ad.datadoghq.com` annotations. 
But I suppose in that case we should have directly a Service and Endpoint object in workloadmetadatastore